lib/nerdtree/notifier.vim

```diff
@@ -15,7 +15,12 @@ function! s:Notifier.NotifyListeners(event, path, nerdtree, params)
     let event = g:NERDTreeEvent.New(a:nerdtree, a:path, a:event, a:params)
 
     for listener in s:Notifier.GetListenersForEvent(a:event)
-        call {listener}(event)
+        if type(listener) == v:t_func
```

Comment:
`listener` needs to be capitalized here and in line 17. Replace the `if` block with this ternary operator assignment.
```vim
    for Listener in s:Notifier.GetListenersForEvent(a:event)
        let Callback = type(Listener) == v:t_func ? Listener : function(Listener)
```